lib/src/components/chat/ds_message_bubble.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:blip_ds/src/theme/systems_colors.dart';
+import 'package:flutter/material.dart';
+
+class DSMessageBubble extends StatelessWidget {
+  final bool alignRight;
+  final Widget child;
+  final EdgeInsets contentPadding;
+  final Map<String, bool> borderRadius;
```

Comment:
Here I think it's better to use an enum instead of Map. With an enum it's easy to know what options I have. With a map of string I must go inside the component to know the exactly the name of options. The enum can be like "Border.topLeft, Border.allLeft, Border.bottomRight, Border.allRight, Border.all...". Or if you want to use a Map, use a Map<Enum, bool>, and this enum with same options you already have (topLeft, bottomLeft...)

Reply:
I'd go with the first suggestion of having an enum that contains all the available options. I think it's more readable than a map.